src/test/java/org/kiwiproject/test/mongo/MongoServerTestsTest.java

```diff
@@ -31,6 +32,12 @@ void shouldStartInMemoryMongoServer() {
         assertThat(mongoServer).isNotNull();
     }
 
```

Comment:
Suggest adding a test that verifies the no-args version uses Mongo 3.6.

Reply:
Not sure how to do that since I can't get access back to the version.

Reply:
You added a `@Getter` on the `serverVersion` field, right?

Reply:
Oh sorry, it's not the _extension_, it's the `MongoServer`...

Reply:
Of course, we could use reflection, but let's not and say we did. Oh, and the 80s are calling me...gotta go! Resolving this conversation as "can't fix" :smirk: